src/main/cpp/bgjs/BGJSV8Engine.cpp

```diff
@@ -572,7 +571,7 @@ v8::Local<v8::Function> BGJSV8Engine::makeRequireFunction(std::string pathName)
                                 String::NewFromOneByte(_isolate, (const uint8_t *) szJSRequireCode,
                                                        NewStringType::kNormal).ToLocalChecked(),
                                 &origin
-                        ).ToLocalChecked()->Run()
+                        ).ToLocalChecked()->Run(context).ToLocalChecked()
```

Comment:
Is it really ok to do this here without checking before? Didnt we change that yesterday? Or was that something else